x-pack/plugins/fleet/public/components/package_policy_delete_provider.tsx

```diff
@@ -201,7 +221,7 @@ export const PackagePolicyDeleteProvider: React.FunctionComponent<Props> = ({
                 id="xpack.fleet.deletePackagePolicy.confirmModal.affectedAgentsMessage"
                 defaultMessage="Fleet has detected that {agentPolicyName} is already in use by some of your agents."
                 values={{
-                  agentPolicyName: <strong>{agentPolicy?.name}</strong>,
+                  agentPolicyName: <strong>{agentPolicies[0]?.name}</strong>,
```

Comment:
Could we display all policies using it here?

Reply:
Yes I missed that! I'm just worried that if the policies are too many the list could become very long.

Reply:
thinking loud here maybe we could have something that looks like a truncate take the first 2 or 3, then show `...` and wrap this in a tooltip that display all policies

Reply:
I added a more generic text and a tooltip with the list of the policies name:
`Fleet has detected that the related agent policies are already in use by some of your agents.`

![Screenshot 2024-06-19 at 10 17 56](https://github.com/elastic/kibana/assets/16084106/82c1cdf1-b83e-49d0-8b84-273c2e102f64)